weave

```diff
@@ -48,7 +48,7 @@ weave launch        [--password <pass>] [--trusted-subnets <cidr>,...]
                     [--no-restart] [--resume] [--no-discovery] [--no-dns]
                     [--ipalloc-init <mode>]
                     [--ipalloc-range <cidr> [--ipalloc-default-subnet <cidr>]]
-                    [--plugin]
+                    [--plugin[=false] --proxy[=false]]
```

Comment:
I think this makes more sense as `[--plugin=false] [--proxy=false]`
Currently it says you can have both or neither, and it doesn't make sense to add without `=false`.